sdk/js/src/service/sessions.js

```diff
@@ -23,10 +23,13 @@ class Sessions {
     autoBind(this)
   }
 
-  async getAllSessions(userId) {
-    const result = await this._api.UserSessionRegistry.List({
-      routeParams: { 'user_ids.user_id': userId },
-    })
+  async getAllSessions(userId, params) {
+    const result = await this._api.UserSessionRegistry.List(
+      {
+        routeParams: { 'user_ids.user_id': userId },
+      },
+      { ...params },
```

Comment:
```suggestion
      params,
```
Please make sure you understand when spreading objects is required. Typically in two situations:
1. You want to compose a new object consisting of data from multiple objects 
2. You want to shallow clone an object (usually to avoid the original object from being modified)